.github/workflows/validate-pipelines.yml

```diff
@@ -26,21 +26,6 @@ jobs:
                 gocd:
                   - 'gocd/**'
 
-    render:
```

Comment:
The validate step renders the YAML and checks the render can be accepted by GoCD.

This action was only needed to check that the committed pipelines matched the rendered pipelines, which is no longer needed since we don't need to commit the pipelines.